Classes/Private/Manager/OCTManager.m

```diff
@@ -243,6 +266,21 @@ - (void)saveTox
     }
 }
 
+#pragma mark - OCTToxDelegate
+
+- (void)toxDidIterate:(OCTTox *)tox
```

Comment:
Do we really want to update file progress every 50ms? Maybe it would be better to have some timer on OCTSubmanagerFiles with more suitable update interval? This way there will be no need to expose didIterate method from OCTTox.